documentation/Using-Azure-TestFramework.md

```diff
@@ -76,7 +76,7 @@ To use this option, set the following environment variable before starting Visua
 `tenantId`: The [AAD Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
 
 
-	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};ServicePrincipalSecret={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
+	TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={SubId};ServicePrincipal={clientId};Password={clientSecret};AADTenant={tenantId};Environment={env};HttpRecorderMode=Record;
```

Comment:
@shahabhijeet I'm not sure I follow.  Is this not the recommended way of running tests: [Using-Azure-TestFramework](https://github.com/Azure/azure-powershell/blob/preview/documentation/Using-Azure-TestFramework.md)?

I followed that guide and created a web app and I am using the ServicePrincipal+Key for authentication in the tests.  It is important to note that **I am not using a username+password**. I am using a ServicePrincipal and ServicePrincipalSecret, but specifying ServicePrincipal+ServicePrincipalSecret does not work for me.

If this is not the right way to do things, what guide should I be following?